tests/python/mkl/test_subgraph.py

```diff
@@ -767,7 +773,7 @@ def test_pos_conv_bn_sum_act():
               "softrelu": True,
               "relu6": False,
               "leakyrelu": True,
-              "gelu": True}
+              "gelu": False}
```

Comment:
why change it false?

Reply:
Convolution with post sum and activation only supports "relu" currently, previously UT didn't catch such failure.